pycbc/io/hdf.py

```diff
@@ -1028,41 +1028,38 @@ def to_coinc_hdf_object(self, file_name):
         # Some fields are special cases
         logging.info("Outputting search results")
         time = self.get_end_time()
-        ofd.create_dataset('time', data=time, dtype=np.float32)
+        ofd['time'] = time
```

Comment:
as for other fields, no need to define a variable which is only used once

Reply:
ok .. maybe put a brief inline comment on that, eg `time will be used to determine active ifos`

Reply:
```suggestion
        # time will be used later to determine active ifos
        ofd['time'] = time
```